config/initializers/devise.rb

```diff
@@ -133,7 +133,7 @@
   config.send_email_changed_notification = true
 
   # Send a notification email when the user's password is changed.
-  config.send_password_change_notification = true
+  config.send_password_change_notification = false
```

Comment:
I thought we wanted this to be true?

Reply:
this is not when we request to change the password, this is when a new password has been set. For the moment we do not have a template for that, but we should probably add it back.